features/directory_structure.feature

```diff
@@ -67,6 +67,19 @@ Feature: Directory Structure
   configured `infer_spec_type_from_file_location!`, RSpec will automatically
   include the correct support functions for each type.
 
+  **Note:** `infer_spec_type_from_file_location!` affects only **the fixed list of folders**: controllers, helpers, jobs, mailers, models, requests, integration, api, routing, views, features.
+
+  To enable automatic metadata for other folder do the next:
+
+  ```ruby
+  RSpec.configuration do |config|
+    # mark all serializer specs with type :serializer
+    config.define_derived_metadata(:file_path => %r{/spec/serializers/}) do |metadata|
```

Comment:
Cucumber doesn't like this for some reason, try running this feature file locally and then correcting the syntax until it works (I think it may be the use of %r{})